### PR TITLE
Add minimal support for creature isle meshes and textures

### PR DESCRIPTION
**Pull request by raffclar**

### CODE REVIEW

**Review by bwrsandman — CHANGES_REQUESTED** [review]
Could you add to the PackFIle.cpp doc at the top of the the source file

**Comment by raffclar** [comment]
> PackFIle
For the PackFile, there's nothing to document but the L3DFile has some minor changes in CI, noticeably to do with DXT5 textures being present. Want me to document that?

**Comment by bwrsandman** [comment]
Yeah a little information about the format would be a useful place to answer questions about the types of textures supported and such.

**Comment by raffclar** [comment]
> Yeah a little information about the format would be a useful place to answer questions about the types of textures supported and such.

Nevermind the key change is in the PackFile

File: components/pack/src/PackFile.cpp [hunk 1]
```diff
@@ -282,10 +285,28 @@ void PackFile::ExtractTexturesFromBlock()
 		DdsHeader ddsHeader;
 		ddsStream.read(reinterpret_cast<char*>(&ddsHeader), sizeof(DdsHeader));
 
-		// TODO(bwrsandman) the extra sizeof(uint32_t) is unaccounted for
-		if (header.ddsSize - sizeof(ddsHeader) - sizeof(uint32_t) != ddsHeader.pitchOrLinearSize)
```
Comment (raffclar) [inline components/pack/src/PackFile.cpp:286]: @bwrsandman 
I'm still reading this example but I don't see any checks like that here: https://docs.microsoft.com/en-us/windows/uwp/gaming/complete-code-for-ddstextureloader
I've had to remove this check as the ddsSize field is larger than the pitchOrLinearSize for DXT5 textures.

**Pull request merged**